src/request.ts

```diff
@@ -179,16 +179,21 @@ async function startFetchRequest(
 
   let res: Response;
 
-  // Timeout after 5 minutes without a response
+  // Timeout after 10 minutes without a response
   //
   // node-fetch's default timeout is 0 and jsforce consumers can't set this when calling `Connection` methods so we set a long default at the fetch wrapper level.
-  const fetchTimeout = options.timeout ?? 300_000
+  const fetchTimeout = options.timeout ?? 600_000
 
   try {
     res = await executeWithTimeout(fetchWithRetries, fetchTimeout, () =>
       controller.abort(),
     );
   } catch (err) {
+    if (err instanceof AbortError) {
+      emitter.emit('error', {
+        message: 'Request was aborted due to timeout of 10 minutes.',
+      })
+    }
```

Comment:
emitting events is will not surface in CLI.
Could we instead append this msg to the one in the error? like `err.message += 'Request was...'`